marketing-checklist.md

```diff
@@ -94,6 +94,12 @@ permalink: /marketing-checklist/
   - [ ] [Kiss Metrics](https://www.kissmetrics.com/)
   - [ ] [Webmaster Tools](https://www.google.com/webmasters/tools)
 
+- [ ] Set up on-site messaging to capture leads and engage customers as they land on your site.
```

Comment:
Excellent idea. I initially included this step in the Optimizations section below because it's not something I'm super experienced with, but I think it deserves its own little section here.

Could you also remove [this line](https://github.com/karllhughes/side-project-marketing/pull/15/files#diff-86d0d98e6cdad9d8067571cac5016058R379) and add [Intercom](https://www.intercom.com/) to your list?

Thanks!